bindings/python/iota_sdk/wallet/wallet.py

```diff
@@ -702,18 +702,21 @@ def announce_candidacy(self, account_id: HexStr) -> BlockId:
         ))
 
     def send_transaction(
-            self, outputs: List[Output], options: Optional[TransactionOptions] = None) -> TransactionWithMetadata:
+            self, immutable_outputs: Optional[List[Output]] = None, mutable_outputs: Optional[List[Output]] = None, options: Optional[TransactionOptions] = None) -> TransactionWithMetadata:
         """Send a transaction.
         """
-        return self.prepare_transaction(outputs, options).send()
+        return self.prepare_transaction(immutable_outputs, mutable_outputs, options).send()
 
     def prepare_transaction(
-            self, outputs: List[Output], options: Optional[TransactionOptions] = None) -> PreparedTransaction:
+            self, immutable_outputs: Optional[List[Output]] = None, mutable_outputs: Optional[List[Output]] = None, options: Optional[TransactionOptions] = None) -> PreparedTransaction:
         """Prepare transaction.
+            Immutable outputs are not allowed to be changed in any way.
+            Mutable outputs may have amount/mana changes made to fulfill requirements.
```

Comment:
> Mutable outputs may have amount/mana changes made to fulfill requirements.
The amount can only increase and this can only happen if the address is controlled by the sender, right?

Reply:
no, the amount of chains can also be reduced

Reply:
well actually not 100% sure the conditions for that reduction so maybe

Reply:
Would be good to have this cleared up (unless we go another route without the mutable outputs)